docs/src/main/asciidoc/logging.adoc

```diff
@@ -77,6 +77,32 @@ class SimpleBean {
 
 NOTE: The logger instances are cached internally. Therefore, a logger injected e.g. into a `@RequestScoped` bean is shared for all bean instances to avoid possible performance penalty associated with logger instantiation.
 
+=== Logging with Panache
```

Comment:
If this is the simpler option, perhaps we should move it up and recommend it against the traditional injection?

Reply:
Good point, I'll move it up.

Reply:
Done.